lib/modules/shared/services/DigitalTwinExporter.ts

```diff
@@ -25,33 +25,33 @@ export class DigitalTwinExporter extends AbstractExporter {
   }
 
   async sendExport(engineId: string, exportId: string) {
-    try {
-      const { query, sort } = await this.getExport(engineId, exportId);
-
-      const digitalTwins = await this.sdk.document.search<DigitalTwinContent>(
-        engineId,
-        this.target,
-        { query, sort },
-        { lang: "koncorde", size: 200 }
-      );
-
-      const namedMeasures = await this.getNamedMeasures(engineId);
-      const measureColumns = await this.generateMeasureColumns(namedMeasures);
-
-      const columns: Column[] = [
-        { header: "Model", path: "_source.model" },
-        { header: "Reference", path: "_source.reference" },
-        ...measureColumns,
-        { header: "lastMeasuredAt", path: "_source.lastMeasuredAt" },
-      ];
-
-      const stream = this.getExportStream(digitalTwins, columns);
-      await this.sdk.ms.del(this.exportRedisKey(engineId, exportId));
-
-      return stream;
-    } catch (error) {
-      this.log.error(error);
-    }
+    const {
+      query,
+      sort,
+      lang = "elasticsearch",
```

Comment:
lang was set to koncorde by default before, we should keep the same otherwise it's a breaking change

Reply:
It's true, but all other requests use `lang='elasticsearch'` by default, so it's inconsistent

Reply:
Doesn't it already default to elasticsearch ? [See this](https://github.com/kuzzleio/kuzzle/blob/4e51e8cf5b3a0efcda0fc73416a5909cb4d01eb3/lib/api/request/kuzzleRequest.ts#L391C58-L391C58)